prusti-viper/src/encoder/high/lower/ty.rs

```diff
@@ -33,6 +33,8 @@ impl IntoPolymorphic<vir_poly::Type> for vir_high::Type {
             vir_high::Type::FunctionDef(ty) => vir_poly::Type::TypedRef(ty.lower(encoder)),
             vir_high::Type::Projection(ty) => vir_poly::Type::TypedRef(ty.lower(encoder)),
             vir_high::Type::Unsupported(ty) => vir_poly::Type::TypedRef(ty.lower(encoder)),
+            vir_high::Type::Lifetime(_ty) => unimplemented!(),
```

Comment:
The lifetimes should be ignored when lowering into `vir_poly`.

Reply:
ignored as in `unreachable!()`?

Reply:
Yes. And also need to ensure that it actually is not called.